tests/Application/config/packages/test/mailer.yaml

```diff
@@ -0,0 +1,5 @@
+framework:
```

Comment:
Remove

Reply:
This can not removed.. It is used to define the service passed as argument on some Sylius services about mailer senders